include/grpcpp/ext/gcp_observability.h

```diff
@@ -17,25 +17,76 @@
 #ifndef GRPCPP_EXT_GCP_OBSERVABILITY_H
 #define GRPCPP_EXT_GCP_OBSERVABILITY_H
 
+#include <grpc/support/port_platform.h>
+
 #include "absl/status/status.h"
+#include "absl/status/statusor.h"
 
 namespace grpc {
-namespace experimental {
+
+namespace grpc_gcp {
```

Comment:
I think if we name it `grpc_gcp` we should probably just drop the top-level namesapce.

Reply:
Craig suggestion might make sense. Either would work.

Having a different namespace would be helpful because there is a clear boundary between vanilla gRPC and GCP specific extensions especially those extensions require an additional dependency. I hope that we can also put `GoogleDefaultCredentials` under gcp namespace but we may have to keep this for backward-compatibility issue. (or we can have both)

Reply:
Moved to `grpc::GcpObservability::Init` as discussed